python_examples/megno/problem.py

```diff
@@ -32,7 +32,7 @@ def setup_planet(com, mass, period, M, omega, eccentricity):
     if eccentricity>0.8:
         E = math.pi
     F = E - eccentricity*math.sin(M) - M
-    for i in xrange(100):
+    for i in range(100):
```

Comment:
Yeah, but that's unrelated to breaking backward compatibility, right?

Incidentally, and interestingly, `range()` is slower than `xrange()` for initializing arrays but slightly faster for indexing.